test/spec/modules/rubiconBidAdapter_spec.js

```diff
@@ -1990,6 +1990,19 @@ describe('the rubicon adapter', function () {
           expect(spec.isBidRequestValid(bidderRequest.bids[0])).to.equal(false);
         });
 
+        it('should be invalid if bidRequest.mediaTypes.video.context is instream but size_id is not defined', function () {
```

Comment:
where is the check for this exactly? I do not see `size_id` being checked if instream anywhere.

Reply:
This test used to be under the hasVideoType tests, but it doesn't actually have anything to do with that function. I just moved it to where a bunch of other spec.isBidRequestValid situations were being tested. No idea if it tests what it is supposed to.